src/services/multiversx-communication/mx.api.service.ts

```diff
@@ -167,11 +168,7 @@ export class MXApiService {
             if (!isNftCollection(collection)) {
                 return undefined;
             }
```

Comment:
maybe a better name here is isCollectionWithoutDecimals, checkNftCollection doesn't say exactly what does it check

Reply:
done